<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientSelectDialogWindow</class>
 <widget class="QDialog" name="PatientSelectDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="FilterLabel">
         <property name="text">
          <string>Filter by:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PhysicianFilterComboBox">
         <item>
          <property name="text">
           <string>Physician...</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="FollowupStatusFilterComboBox">
         <item>
          <property name="text">
           <string>Followup Status...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Overdue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Received</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Completed</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FilterPushButton">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="PatientListWidget"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="SelectCancelHorizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SelectPushButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="StatusLabel">
     <property name="text">
      <string>Select a Patient.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
